.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,7 @@
 <!--
 1. Please check out and follow our Contributing Guidelines: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md
-2. Fill out the whole template so we have a good overview on the issue
-3. Do not remove any section of the template. If something is not applicable leave it empty but leave it in the PR
-3. Please follow the template, otherwise we'll have to ask you to update it
+2. Do not remove any section of the template. If something is not applicable leave it empty but leave it in the PR
```

Comment:
Yes, for simply typo stuff people should leave everything empty and just say typo. But simple typo PRs are so few compared to others that we shouldn't optimise towards them. And the biggest problem with removing stuff from the template is that we have to think "So this doesn't look like it includes everything, what was removed, ok let me compare with something else that we don't forget anything" .

This PR template list is as much a list for us as it is for the users, e.g. if they remove the "write documentation" item, we might forget to look or ask for it as well.